xapian-applications/omega/handler_libebook.cc

```diff
@@ -0,0 +1,133 @@
+/** @file handler_libebook.cc
+ * @brief @brief Extract text and metadata using libe-book.
```

Comment:
Double `@brief`.

I see this is was also present in the merge changes for poppler, so I pushed a fix for those: a600cbb9db8424e29d5db2b91a1e5885329127ef

Reply:
Sorry, I didn't see that.